test-case/verify-pcm-list.sh

```diff
@@ -26,16 +26,16 @@ OPT_HAS_ARG['t']=1         OPT_VAL['t']="${TPLG:-}"
 func_opt_parse_option "$@"
 tplg=${OPT_VAL['t']}
 
-tplg_path=`func_lib_get_tplg_path "$tplg"`
-[[ "$?" != "0" ]] && die "No available topology for this test case"
+tplg_path=$(func_lib_get_tplg_path "$tplg") \
+	|| die "No available topology for this test case"
```

Comment:
You don't need to escape the new line when you do this:
```shell
tplg_path=$(func_lib_get_tplg_path "$tplg") ||
	die "No available topology for this test case"
```
In many cases when you leave something incomplete the shell knows it should keep reading and then you don't need to escape the new line.